Calling/ClientApp/src/components/ChatThread.tsx

```diff
@@ -322,6 +343,30 @@ export default (props: ChatThreadProps): JSX.Element => {
             <Chat
               styles={chatStyle}
               items={messagesWithAttached.map((message: any, index: number) => {
+                if (isUploadedFileMessage(message.content)) {
+                  const messageContent = JSON.parse(message.content);
+                  if (!messageContent) {
+                    return {};
+                  }
+
+                  return {
+                    key: index,
+                    contentPosition: message.mine ? 'end' : 'start',
```

Comment:
Should the file card be rendering on the left/right depending on who sent it? When I run it, file messages are always rendered on the left side of the chat.

Reply:
yeah, i noticed this too - might be because there is no "mine" property on this message - perhaps we can add this is a backlog item?